src/views/project/duplication/RequestView.vue

```diff
@@ -0,0 +1,97 @@
+<script setup>
+import { computed, ref, onMounted } from 'vue'
+import { useRoute } from 'vue-router'
+import router from '@/router'
+import { useProjectDuplicationStore } from '@/stores/ProjectDuplicationStore'
+import { useProjectUsersStore } from '@/stores/ProjectUsersStore'
+import { useUserStore } from '@/stores/UserStore'
+import LoadingIndicator from '@/components/project/LoadingIndicator.vue'
+
+const route = useRoute()
+const projectId = route.params.id
+const message = ref([null])
+let transfer = ref(false)
+const ProjectDuplicationStore = useProjectDuplicationStore()
+const isLoaded = computed(() => ProjectDuplicationStore.isLoaded)
+
+onMounted(() => {
+  if (!ProjectDuplicationStore.isLoaded) {
+    ProjectDuplicationStore.checkForConditions(projectId)
+  }
+})
+
+async function makeRequest(remark) {
+  if (remark == null || remark.replace(/\s/g, '').length == 0) {
+    alert(
+      'There were errors in your form: Please include the reason for the request'
+    )
+  } else {
+    ProjectDuplicationStore.sendRequest(projectId, remark, transfer.value)
+    router.push({ path: `/myprojects/${projectId}/overview` })
+  }
+}
+</script>
+
+<template>
+  <LoadingIndicator :isLoaded="isLoaded">
+    <header>Project: P{{ projectId }} Test</header>
+
+    <RouterLink :to="`/myProjects/${projectId}/overview`">
+      <button class="action-bar">Back to Project Overview</button>
+    </RouterLink>
+
+    <p>
+      Please use the following form to submit a request to duplicate P{{
+        projectId
+      }}. After submission, a MorphoBank administrator will contact you with
+      further information.
+    </p>
+
+    <div v-if="ProjectDuplicationStore.onetimeMedia.length > 0">
+      <header>
+        There are media licensed for onetime use in this project. This license
```

Comment:
`There are {{ ontimeMedia.length }} media`

We put the number of media that affected.